### DISCUSSION TITLE
Label management

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#21012] [Discussions] [answered]
How to add, remove or manage labels of your discussion?

**Comment by jfullstackdev** [comment, marked as answer]
go to 

![image](https://user-images.githubusercontent.com/47092464/180435267-0de77728-eb62-422c-afe6-3a48b4ca1cae.png)

click `Edit labels` then 

either add or edit 

![image](https://user-images.githubusercontent.com/47092464/180435334-4b2514ac-d270-4648-a823-b3c20c22cde6.png)
